### ISSUE TITLE
Setting nginx credentials failed, remained default

### ISSUE THREAD

**Issue by svollath** [SUSE/stratos-metrics#30]
I deployed metrics to CaaSP3 as well as to Azure AKS - in both cases the nginx credentials remained default:

```
> kubectl logs stratos-metrics-cfdevio-fcdc877bb-55mkm -n metrics
Checking if certificate has been supplied!
TLS certificate detected continuing, starting nginx.
Create username/password for authentication
Adding password for user metrics
Starting nginx
```

while my metrics-config-values.yaml contained (e.g. azure):

```
secrets:
  CLUSTER_ADMIN_PASSWORD: password
  UAA_ADMIN_CLIENT_SECRET: password
env:
  DOMAIN: 40.68.160.43.omg.howdoi.website
  UAA_HOST: uaa.40.68.160.43.omg.howdoi.website
  UAA_PORT: 2793
  GARDEN_ROOTFS_DRIVER: "overlay-xfs"
kube:
  external_ips: ["10.240.0.6", "10.240.0.5", "10.240.0.4"]
  storage_class:
    persistent: "default"
    shared: "shared"
  registry:
    hostname: "registry.suse.com"
    username: ""
    password: ""
  organization: "cap"
  auth: "rbac"
  psp:
    nonprivileged: "suse.cap.psp"
    privileged: "suse.cap.psp"
---
kubernetes:
  authEndpoint: https://svollath-svollath-cap-aks-f416cc-f0ebd67d.hcp.westeurope.azmk8s.io:443
prometheus:
  kubeStateMetrics:
    enabled: true
  imagePullSecrets:
  - name: regsecret
nginx:
  username: admin
  password: password
```

```
helm install suse/metrics --name metrics --namespace metrics -f metrics-config-values.yaml
```

**Comment by svollath** [comment]
At least, when I remove the "---" from my metrics-config-values.yaml it works. So it seems values from the second file section are ignored.

I was referring to our official documentation, that suggests to split the config into two files:

```
> helm install suse/metrics \
    --name susecf-metrics \
    --namespace metrics \
    --values scf-config-values.yaml \
    --values stratos-metrics-values.yaml
```

and then I just thought, that having one file where the sections are separated with "---" is equivalent - but it doesn't seem so.

**Comment by svollath** [comment]
Yes, so this is a pilot error (or helm issue).
However, since I've been on the path of testing setting the nginx port, I've came across two documentation issues.
First, the default nginx port is not 7443 like documented here, but 443. If that's the planned fix for metrics-1.0.1 - having 7443 default, to not interfere with 443 - then I can confirm that e.g. on Azure AKS, everything works as expected, after restarting an Azure cluster, when metrics is configured for 7443.
Second is that our default CAP documentation suggests to have two config-values.yaml files - the scf-config-values.yaml and the stratos-metrics-values.yaml (see comment above).
It would be good, if the documentation here (Readme.md) would not only suggest to reuse scf-config-values.yaml, but also give an example on how to "helm install" with two values files given.
Also, users following the CAP documentation will have the "suse" repo from suse kubernetes charts, so would expect to install the same chart from suse/metrics.

**Comment by nwmac** [comment]
@svollath Thanks - I was just looking into it - the --- is the cause of the problem as you discovered.

1.0.1 will fix the port problem. I'm just testing that now.
